crm_security_group/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "CRM Only Security Groups",
+    "summary": "Add new group in Sales to show only CRM",
+    "version": "13.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "https://github.com/OCA/crm/",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["crm", "sale_crm", "sale_management"],
```

Comment:
Why do you need to depend on `sale_crm` and `sale_management`? That shouldn't be correct.